src/main/java/com/github/manolo8/darkbot/config/ConfigHandler.java

```diff
@@ -120,4 +128,13 @@ public void onPluginUnload(ExtensionsAPI.PluginLifetimeEvent e) {
         }
     }
 
+    @Override
+    public Collection<String> options() {
+        return Stream.concat(Stream.of("(none)"), loader.getAvailableConfigs().stream()).collect(Collectors.toList());
```

Comment:
none should not be an option, the option is `null`, and it's `getText` that should return "(none)" for the null case

Reply:
reverted